infrastructure/variables.tf

```diff
@@ -62,6 +62,14 @@ variable "common_config" {
   })
 }
 
+variable "common_infra_config" {
+  description = "Config for the common infra"
+  type = object({
+    common_infra_vnet_name = string
+    common_infra_rg        = string
```

Comment:
The config files have `vnet_name` and `network_rg` - so we need to make sure they all match.